src/main/java/com/apeshko/springboot/service/TicketsService.java

```diff
@@ -0,0 +1,13 @@
+package com.apeshko.springboot.service;
+
+import com.apeshko.springboot.model.Ticket;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface TicketsService {
+    List<Ticket> findAll();
+    Optional<Ticket> findById(int id);
+    Ticket save(Ticket ticket);
+    void deleteById(int id);
```

Comment:
good practice to return deleted entity (or at least id)

Reply:
@mikalai-amusin followed JPA interface, but agree that it makes sense. thx.